web_edit_user_filter/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+* Make individual values in facets editable.
```

Comment:
> > 1. When user hits the pencil button and edits the filter, the only way to update it later is deleting the previous one and saving it again with the same name. Some "update" button, or the ability to save it with the same name again and trigger that as an update, would be great.
> 
> I've added 1 to the roadmap

I think you didn't get the point :thinking:. I was referring to the fact that you cannot update a favorite filter; you have to delete it and save it again instead:


![peek 14-02-2019 08-40](https://user-images.githubusercontent.com/973709/52774297-ca2ead00-3034-11e9-8102-c904c96948a5.gif)

This fact is a little bit confusing for a new user, so it should be probably documented here.

Making individual values in facets editable would be good, but the fact that you can only remove them and you can add a new one using normal odoo search is, instead, very straightforward and quick to understand with the UI you designed. :blush:

Reply:
I understand now :smile:

I have some ideas to do this:
1. The pencil icon changes into a floppy disk icon (or can be added next to it) for the filter that is being edited. If it's pressed the filter will be overwritten.
2. or with a checkbox 'Overwrite' in "Save current search" which overwrites the filter if the name already exists.

Reply:
I like 1 more, what do you think?